addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -489,6 +489,20 @@
             <field name="act_window_id" ref="hr_expense_actions_my_all"/>
         </record>
 
+        <record id="view_product_hr_expense_form" model="ir.ui.view">
+            <field name="name">product.template.expense.form</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_form_view"/>
+            <field name="arch" type="xml">
+                <div name="options" position="inside">
+                    <div>
```

Comment:
should be a `<span class="d-inline-block">` for consistency with the other elements inside the `<div name="options">`.